ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -330,6 +330,7 @@ export class AdminInstitutionEditImpl extends React.Component<UrlParamsProps, In
                 inputStyle={{width: '16rem', marginTop: '0.3rem'}}
                 labelText='Institution Name'
                 onChange={v => this.setState(fp.set(['institution', 'displayName'], v))}
```

Comment:
Attempting to trim for `onChange` immediately removes all trailing spaces, making it impossible to type one.

`onBlur` fixes it in both cases: the user immediately hits Save or moves on to a different input component.